src/target/drivers/display/8080/320x240x16.c

```diff
@@ -83,22 +125,23 @@ void LCD_DrawPixelXY(unsigned int x, unsigned int y, unsigned int color)
 
 void LCD_DrawStart(unsigned int x0, unsigned int y0, unsigned int x1, unsigned int y1, enum DrawDir dir)
 {
-  if (dir == DRAW_SWNE) {
-    unsigned int y = 239 - y0;
-    y0 = 239 - y1;
-    y1 = y;
-    screen_flip = 1;
-  } else {
-    screen_flip = 0;
-  }
-  //printf("LCD_DrawStart: (%d, %d) - (%d, %d)\n", x0, y0, x1, y1);
-  disp_type->draw_start(x0, y0, x1, y1);
-  return;
+    if (dir == DRAW_SWNE) {
```

Comment:
we should move logic of dir to other files. it doesn't make a clear abstraction for a driver to understand draw direction.